client/app/scripts/liveblog-edit/module.js

```diff
@@ -243,6 +243,10 @@ define([
         $scope.$watch(angular.bind(this, function () {return this.blogPreferences;}), function(new_value) {
             vm.isSaved = _.isEqual(vm.blogPreferences, vm.blog.blog_preferences);
         }, true);
+        _.templateSettings.interpolate = /{{([\s\S]+?)}}/g;
+        var compiled = _.template(_.trim(document.getElementById('liveblog-embed-template').innerHTML));
```

Comment:
It's hidden in the DOM, but it's still a DOM element. Anyway, this kind of operation like `angular.element()` or `($)document.getElementById` shouldn't be done in the controller but in a directive if we want to follow the angular principles.